docs/_posts/2017-07-26-error-handling-in-react-16.md

```diff
@@ -55,7 +55,13 @@ The `componentDidCatch()` method works like a JavaScript `catch {}` block, but f
 
 Note that **error boundaries only catch errors in the components below them in the tree**. An error boundary can’t catch an error within itself. If an error boundary fails trying to render the error message, the error will propagate to the closest error boundary above it. This, too, is similar to how `catch {}` block works in JavaScript.
 
+## Stack traces
+
+By default Error boundaries stack traces would look like this:
```

Comment:
Let's add some context here? It's not clear what stack traces this refers to. Something like:

"React 16 prints all errors to the console in development, even if the application accidentally swallows them. In addition to the error message and the JavaScript stack, it also now provides component stacks. Now you can see where exactly in the component tree the failure has happened:"